waffle-optimism/package.json

```diff
@@ -4,12 +4,8 @@
   "version": "4.0.2",
   "author": "Marek Kirejczyk <account@ethworks.io> (http://ethworks.io)",
   "repository": "git@github.com:EthWorks/Waffle.git",
-  "private": false,
+  "private": true,
   "license": "MIT",
-  "publishConfig": {
```

Comment:
Why? The package exports `OptimismProvider`

Reply:
Rolledback